### ISSUE TITLE
Expose membership type in the Admin UI for organization members

### ISSUE THREAD

**Issue by pedroigor** [keycloak/keycloak#33498]
### Description

For the sake of UX, we should expose the membership type when fetching the representation for users that belong to an organization. It should make it a lot easier to identify whether a user is a `managed` or `unmanaged` organization member and make it more clear how certain operations will impact the user. For instance, unlinking a `managed` member from an organization will remove the account from the realm whereas the account for an `unmanaged` member is kept and only the link to the organization is removed.

The Organization Admin API is already exposing the membership type of a member from the `/admin/realms/{realm}/organizations/{orgid}/members` endpoint when querying members of an organization. There is a `membershipType` property within the member representation.

In the Admin UI, we can probably have a column in the `Members` list to show the membership type. We could also expand the idea and perhaps have some filter to show only `managed` or `unmanaged` members (or all of them).

I'm not sure though if we want or how we want to do that same when managing users through the User UI. 

### Motivation

Improve UX in the Admin UI when managing organization members.

**Comment by ahus1** [comment]
@agagancarczyk - I see you manually added the label 26.0.6. This is added automatically once the backport PR is merged. Instead, add the backport label. 
Relying on the automation ensures that the labels are correctly even if a release is cut from a release branch and those PRs haven't been merged yet.
